.release-please-manifest.json

```diff
@@ -7,6 +7,7 @@
   "co-circom/co-groth16": "0.5.1",
   "co-circom/co-plonk": "0.3.1",
   "co-noir/co-acvm": "0.2.0",
+  "co-noir/co-builder": "0.1.0",
```

Comment:
this is the currently released version, which should be 0.0.1 for this

Reply:
fixed